### PR TITLE
Functionality added to return LinkedList as string with ToString()

### PR DESCRIPTION
**Pull request by CarlHelberg**
ToString added... possibly added more branches to repo

### CODE REVIEW

File: LinkedList.cs [hunk 1]
```diff
@@ -16,24 +16,68 @@ public Node(T nodeInput)
                 next = null;
             }
         }
-        static class LinkedList<T>
+         class  LinkedList<T>
         {
-            public static Node<T> Head;
+            public   Node<T> Head;
+            public   int Length = 0;
```
Comment (A-Helberg) [inline LinkedList.cs:22]: Why do we need the length?

File: LinkedList.cs [hunk 2]
```diff
@@ -16,24 +16,68 @@ public Node(T nodeInput)
...
 
-            public static void Prepend( T prependValue)
+            public  void Prepend( T prependValue)
             {
                 Node<T> PrependThis = new Node<T>(prependValue);
                 PrependThis.next = Head;
                 Head = PrependThis;
-                
+                Length++;
```
Comment (A-Helberg) [inline LinkedList.cs:29]: Same as above

File: LinkedList.cs [hunk 3]
```diff
@@ -16,24 +16,68 @@ public Node(T nodeInput)
...
+                    tempNext = tempValue.next;
+                    toReturn += tempValue.value;
+                    if(tempNext != null)
+                    {
+                        toReturn += ", ";
+                    }
+                    tempValue = tempValue.next; 
+                }
+                toReturn += ")";
+                System.Console.WriteLine(toReturn);
```
Comment (A-Helberg) [inline LinkedList.cs:48]: 👎

File: LinkedList.cs [hunk 4]
```diff
@@ -16,24 +16,68 @@ public Node(T nodeInput)
...
+                return toReturn;
+            }
+
+            public void Traverse()
```
Comment (A-Helberg) [inline LinkedList.cs:52]: Try to implement this to work with foreach

**Review by A-Helberg — CHANGES_REQUESTED** [review]

File: LinkedList.cs (hunk 1 above)
Comment (CarlHelberg) [reply LinkedList.cs:22]: Feature added to return Length without needing to Traverse the whole list.

File: LinkedList.cs (hunk 2 above)
Comment (CarlHelberg) [reply LinkedList.cs:29]: Increment Length to instantly have access to Length property, will also be incremented in Add()/Append() and decremented in Remove().

File: LinkedList.cs (hunk 3 above)
Comment (CarlHelberg) [reply LinkedList.cs:48]: Removed in next commit.

**Review by A-Helberg — CHANGES_REQUESTED** [review]
Remember to implement traverse such that it works with for each

**Pull request merged**